### PR TITLE
Allow datetime to process strings with nanoseconds

### PR DESCRIPTION
**Pull request by cecinestpasunepipe**
(DIS-1964)

### CODE REVIEW

File: tests/test_fieldtypes.py [hunk 1]
```diff
@@ -9,13 +9,9 @@
...
-from flow.record.fieldtypes import (
-    PATH_POSIX,
-    PATH_WINDOWS,
-    fieldtype_for_value,
-    net,
-    uri,
-)
+from flow.record.fieldtypes import PATH_POSIX, PATH_WINDOWS
+from flow.record.fieldtypes import datetime as dt
+from flow.record.fieldtypes import fieldtype_for_value, net, uri
```
Comment (pyrco) [inline tests/test_fieldtypes.py:14]: Why this change? isort or black don't like the tuple formatting when `as` is used?

Comment (cecinestpasunepipe) [reply tests/test_fieldtypes.py:14]: Yes, that was isort I'm afraid...

**Review by pyrco — APPROVED** [review]
